### ISSUE TITLE
Draw component diagram for Admin module

### ISSUE THREAD

**Issue by Mohit-Chaudhari** [Mohit-Chaudhari/KarmaYoga#1]
**Admin module should contain following facilities**,
1. Admin should be able to manage caste list.
2. Admin should able to manage college department list.
3. Admin should able to approve or disapprove student profile.
4. Admin should able to see all details of every student.
5. Admin should able to view all statistical diagrams according to data.
6. Admin should able to calculate student payment details according to working hours.
7. Student stats and its working hours should able to calculate and displayed on admin dashboard.

Consider above details and draw component diagram for admin module. Please make sure that all components should able to communicate with each other. Consider you are getting details from teacher and student module, draw component diagram accordingly.

@Sonalianildeshmukh 
If you have any suggestions regarding this issue then please updates your thoughts.

**Comment by Sonalianildeshmukh** [comment]
Yes admin should get details from all other module. Scholarship criteria also be visible to admin along with eligible student list.

**Comment by Mohit-Chaudhari** [comment]
@shubhangi-cloud Please consider above scenario mentioned by Sonali ma'am.
